haskell/snapshot.yaml

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: 2021 Arthur Breitman
+# SPDX-License-Identifier: LicenseRef-MIT-Arthur-Breitman
+
+name: segmented-cfmm-snapshot
```

Comment:
Do we need this?

I think its content can be moved to `stack.yaml`

Reply:
Probably not, I moved all this to the `stack.yaml`.